server/cli/annotate.py

```diff
@@ -53,8 +72,8 @@ def wrapper(*args, **kwargs):
 @click.option(
     "-g",
     "--gene-column-name",
-    help="The name of the `var` column that contains gene identifiers. The values in this column will be used to match "
-    "genes between the query and reference datasets. If not specified, the gene identifiers are expected to exist "
+    help="The name of the `var` column that contains gene names. The values in this column will be used to match "
```

Comment:
maybe say ''gene symbols'' instead of ''gene names'' here?

Reply:
settled on keeping `gene names`, since that's in fact what we get from `feature_name` in our schema-compliant h5ad files that are used to train our models (at least for the lung model).